### DISCUSSION TITLE
New condition to Scheduler

### DISCUSSION THREAD

**Discussion by lDoomerl** [Zrips/CMI#9114] [Enhancement]
Add condition "Messenges to repeat" or something like that. This condition will allow you to set the number of player messages in the chat, after which the scheduled task will be repeated. 
It will help to avoid spam of the server announcements especially in the night time, when online is low.

**Comment by CloudeLecaw** [comment]
that sounds more like a chatflter expansion than a schedule...
might be hard.. it would take record over whole chat over time...
